pkg/cassandra/datacenter.go

```diff
@@ -88,9 +93,41 @@ func NewDatacenter(klusterKey types.NamespacedName, template *DatacenterConfig)
 		dc.Spec.Resources = *template.Resources
 	}
 
+	if len(template.MgmtAPIHeap) != 0 {
+		if err := SetMgmtAPIHeap(dc, template.MgmtAPIHeap); err != nil {
+			return nil, err
+		}
+	}
+
 	return dc, nil
 }
 
+// SetMgmtAPIHeap sets the management API heap size on a CassandraDatacenter
+func SetMgmtAPIHeap(dc *cassdcapi.CassandraDatacenter, heapSize string) error {
```

Comment:
In which case does it return err? I don't see any.

Reply:
It doesn't, you're right. Fixed.